src/BIP32.php

```diff
@@ -161,7 +161,7 @@ public static function CKD($master, $address_definition, $generated = array())
         }
 
         if (!isset($data)) {
-            return false;
+            throw new \Exception(); // I don't think this can happen
```

Comment:
yea, it's safe to asume that type will always be there in the `import` result, and currently is either private or public

but I guess we should change the `if (public) {} else if(private) {}` to `if (public) {} else {}` then ...